stores/stores.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/getsops/sops/v3/pgp"
 )
 
+const (
+	// The key used to store SOPS metadata at in SOPS encrypted files.
+	SopsMetadataKey = "sops"
```

Comment:
Nit: start the comment off with the variable name

```suggestion
	// SopsMetadataKey is the key used to store SOPS metadata at in SOPS encrypted files.
	SopsMetadataKey = "sops"
```